html/cross-origin-embedder-policy/blob.https.html

```diff
@@ -0,0 +1,31 @@
+<!doctype html>
+<meta charset=utf-8>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/get-host-info.sub.js"></script>
+<script src="/common/utils.js"></script>
+<div id=log></div>
+<script>
+const origins = get_host_info();
```

Comment:
In this case the frame made from the blob inherits the COEP settings, right? If so, I'd be happier if we actually check that we reject opaque no-corp responses in it. Same for data URLs and iframe srcdoc.